docs/StardustDocs/topics/gradleReference.md

```diff
@@ -10,7 +10,7 @@ dataframes {
     }
 }
 ```
-Note than name of the file and the interface are normalized: split by '_' and ' ' and joined to camel case.
+Note the name of the file and the interface are normalized: split by '_' and ' ' and joined to camel case.
```

Comment:
*Note that*

Reply:
actually _Note that the_ :)

Reply:
Also I usually 'CamelCase' the word for clarity, but that's optional of course ;)